src/ps2dev.cpp

```diff
@@ -9,307 +9,401 @@
-  _PS2DBG.print(F("received data "));
-  _PS2DBG.println(*value,HEX);
-  _PS2DBG.print(F("received parity "));
-  _PS2DBG.print(received_parity,BIN);
-  _PS2DBG.print(F(" calculated parity "));
-  _PS2DBG.println(received_parity,BIN);
+	_PS2DBG.print(F("received data "));
+	_PS2DBG.println(*value, HEX);
+	_PS2DBG.print(F("received parity "));
+	_PS2DBG.print(received_parity, BIN);
+	_PS2DBG.print(F(" calculated parity "));
+	_PS2DBG.println(received_parity, BIN);
 #endif
-  if (received_parity == calculated_parity) {
-    return 0;
-  } else {
-    return -2;
-  }
 
+	return received_parity == calculated_parity ? 0 : -2;
 }
 
 
 void PS2dev::keyboard_init()
 {
-  while(write(0xAA)!=0);
-  delay(10);
-  return;
+	while (write(0xAA) != 0) ;
+
+	delay(10);
 }
 
 void PS2dev::ack()
 {
-  while(write(0xFA));
-  return;
+	while (write(0xFA)) ;
 }
 
 int PS2dev::keyboard_reply(unsigned char cmd, unsigned char *leds)
 {
-  unsigned char val;
-  unsigned char enabled;
-  switch (cmd)
-  {
-  case 0xFF: //reset
-    ack();
-    //the while loop lets us wait for the host to be ready
-    while(write(0xAA)!=0);
-    break;
-  case 0xFE: //resend
-    ack();
-    break;
-  case 0xF6: //set defaults
-    //enter stream mode
-    ack();
-    break;
-  case 0xF5: //disable data reporting
-    //FM
-    enabled = 0;
-    ack();
-    break;
-  case 0xF4: //enable data reporting
-    //FM
-    enabled = 1;
-    ack();
-    break;
-  case 0xF3: //set typematic rate
-    ack();
-    if(!read(&val)) ack(); //do nothing with the rate
-    break;
-  case 0xF2: //get device id
-    ack();
-    write(0xAB);
-    write(0x83);
-    break;
-  case 0xF0: //set scan code set
-    ack();
-    if(!read(&val)) ack(); //do nothing with the rate
-    break;
-  case 0xEE: //echo
-    //ack();
-    write(0xEE);
-    break;
-  case 0xED: //set/reset LEDs
-    ack();
-    if(!read(leds)) ack(); //do nothing with the rate
+	unsigned char val;
+	unsigned char enabled;
+
+	switch (cmd)
+  	{
+  		case 0xFF: //reset
+    		ack();
+
+    		//the while loop lets us wait for the host to be ready
+    		while (write(0xAA) != 0) ;
+			
+    		break;
+  		case 0xFE: //resend
+    		ack();
+    		break;
+  		case 0xF6: //set defaults
+    		//enter stream mode
+    		ack();
+    		break;
+  		case 0xF5: //disable data reporting
+    		//FM
+    		enabled = 0;
+    		ack();
+    		break;
+  		case 0xF4: //enable data reporting
+    		//FM
+    		enabled = 1;
+    		ack();
+    		break;
+  		case 0xF3: //set typematic rate
+    		ack();
+    		if (!read(&val)) ack(); //do nothing with the rate
+    		break;
+  		case 0xF2: //get device id
+    		ack();
+    		write(0xAB);
+    		write(0x83);
+    		break;
+  		case 0xF0: //set scan code set
+    		ack();
+    		if (!read(&val)) ack(); //do nothing with the rate
+    		break;
+  		case 0xEE: //echo
+    		write(0xEE);
+    		break;
+  		case 0xED: //set/reset LEDs
+    		ack();
+    		if (!read(leds)) ack(); //do nothing with the rate
```

Comment:
you reformating my code makes it bit hard to actualy see what changes you made. i think we can agree on using some clang-format config, but changing formating SHOULD be done as separate pull request, so it's obvious that it only changes the style. but i would prefer you to not change the style, or at least provide the .clang-format file.

Reply:
Ok!

Reply:
That makes sense.